.github/workflows/ci.yml

```
@@ -10,13 +10,15 @@ jobs:
       fail-fast: false
       max-parallel: 5
       matrix:
-        python-version: ['3.6', '3.7', '3.8', '3.9', '3.10']
-        django-version: ['2.2', '3.2']
+        python-version: ['3.8', '3.9', '3.10', '3.11', '3.12']
+        django-version: ['3.2','4.2']
```

MAY: 4.1もテストされていると有り難いです。

```suggestion
        django-version: ['3.2','4.1','4.2']
```

3.2から4系に上げる場合、4.2ではMySQL5がサポート対象外になっていることから、一旦4.1に上げるケースがあります。このため、4.1もテストが通って居ると少し気が楽です。

toxでdj41を追加してテストしたところ、コード変更なしでテストは通りました。